### PR TITLE
chore(deps): update stylelint to 15 (VIV-1596)

### PR DESCRIPTION
**Pull request by RichardHelm**

### CODE REVIEW

File: libs/components/src/lib/alert/alert.scss [hunk 1]
```diff
@@ -83,8 +79,8 @@ $alert-transition-delay: --transition-delay;
 	display: flex;
 	align-items: center;
 	padding: 16px;
-	background-color: var(--vvd-color-neutral-50);
 	border-radius: #{border-radius-variables.$border-radius-normal};
+	background-color: var(--vvd-color-neutral-50);
```
Comment (rachelbt) [inline libs/components/src/lib/alert/alert.scss:83]:
```suggestion
	background-color: var(#{constants.$vvd-color-neutral-50});
```
not related to the PR but better like this :)

File: libs/components/src/lib/button/button.scss [hunk 2]
```diff
@@ -34,20 +32,18 @@
...
-	border-radius: var(#{variables.$button-border-radius});
 	box-shadow: inset 0 0 0 1px var(#{appearance.get-appearance-token(outline)});
 	color: var(#{appearance.get-appearance-token(text)});
 	gap: var(#{variables.$icon-gap});
-	text-decoration: none; 
+	text-decoration: none;
 	vertical-align: middle;
 
 	.text {
-		/* stylelint-disable value-no-vendor-prefix */
```
Comment (rachelbt) [inline libs/components/src/lib/button/button.scss:48]: yesssss 👍  :)

File: libs/components/src/lib/time-picker/time-picker.scss [hunk 3]
```diff
@@ -59,15 +60,14 @@
 	.picker {
 		position: relative;
 		display: flex;
+		overflow: hidden hidden;
```
Comment (rachelbt) [inline libs/components/src/lib/time-picker/time-picker.scss:63]: isn't `overflow: hidden;` is enough?

**Review by rachelbt — COMMENTED** [review]
Nice!
maybe we do not need the ignor that I added any more:
https://github.com/Vonage/vivid-3/pull/1561/files

File: libs/components/src/lib/time-picker/time-picker.scss (hunk 3 above)
Comment (RichardHelm) [inline libs/components/src/lib/time-picker/time-picker.scss:63]:
```suggestion
		overflow: hidden;
```

**Comment by RichardHelm** [comment]
> Nice! maybe we do not need the ignor that I added any more: https://github.com/Vonage/vivid-3/pull/1561/files

You're right, removed it now 👍

**Review by rachelbt — APPROVED** [review]
LGTM

**Pull request merged**